Blockzilla/InternalOnboardingSettingsView.swift

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import SwiftUI
+
+struct InternalOnboardingSettingsView {
+    @ObservedObject var internalSettings = InternalSettings()
+}
+
+extension InternalOnboardingSettingsView: View {
+    var body: some View {
+        Form {
+            Toggle(isOn: $internalSettings.alwaysShowOnboarding) {
+                VStack(alignment: .leading) {
```

Comment:
Why is `VStack` necessary?

Reply:
Actually,it is not. It was a reziduu from the similar UI that is already implemented.